### PR TITLE
[CMP] Add pubData for Sourcepoint Reporting

### PR DESCRIPTION
**Pull request by mxdvl**
## Why are you doing this?

In order to join consent signals with browsers, we need to send the page view id to Sourcepoint.

## Changes
* Imported `ophan.viewId`
* Added the `pubData` object to `cmp.init`
* Added `pageViewId` and `platform` params

### CODE REVIEW

**Review by ripecosta — APPROVED** [review]
👍  LGTM but might be best if one of the code owners leaves a review.

**Review by GHaberis — APPROVED** [review]
Looks good to me

**Comment by mxdvl** [comment]
I don’t have merging rights, so if any of you is happy to merge, that would be 💯

**Pull request merged**
